src/main/java/de/tum/cit/aet/artemis/assessment/dto/FeedbackAffectedStudentDTO.java

```diff
@@ -0,0 +1,4 @@
+package de.tum.cit.aet.artemis.assessment.dto;
+
+public record FeedbackAffectedStudentDTO(long courseId, long participationId, String firstName, String lastName, String login, String repositoryName) {
```

Comment:
In later parts, e.g. `StudentParticipationRepository` you are talking about a repositoryURI, is it a Name or an URI?
```suggestion
public record FeedbackAffectedStudentDTO(long courseId, long participationId, String firstName, String lastName, String login, String repositoryURI) {
```

Reply:
its a url saved as a string on server side